jakartaee11/JakartaEE11ReleasePlan.md

```diff
@@ -198,61 +200,60 @@ As stated earlier, all component Specifications which plan a Major or Minor rele
 
 ### Waves
 
-We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications.
+We are proposing to deliver Jakarta EE 11 in a set of waves similar to those delivered in the previous Jakarta EE releases. These waves are somewhat related to the dependency tree of specifications. We aim to deliver specifications with a low number of dependencies first followed by other specifications. (updated specifications marked with asterisks)
 
 #### Wave 1
 
-- Jakarta Annotations
+- Jakarta Annotations*
 
 #### Wave 2
 
 - Jakarta Expression Language
-- Jakarta Interceptors
+- Jakarta Interceptors*
 - Jakarta Lang Model (may be released with Jakarta CDI [Core] in wave 3)
 
 #### Wave 3
 
-- Jakarta Activation
-- Jakarta Contexts and Dependency Injection
+- Jakarta Activation*
```

Comment:
Should Activation be removed from this list since it  isn't being updated for EE 11?

Reply:
I have the same question for other specs that are listed but not changing for EE 11.  I'll wait to here the answer for ^ before commenting on the others.

Reply:
I guess Activation is special as we are only doing a service release for that.  I'll check the others that I thought were wrong.  Deleting these comments soon.